RELEASE/scripts/autoscend/quests/level_10.ash

```diff
@@ -11,8 +11,8 @@ boolean L10_plantThatBean()
 	string page = visit_url("place.php?whichplace=plains");
 	if(contains_text(page, "place.php?whichplace=beanstalk"))
 	{
-		auto_log_warning("I have no bean but I see a stalk here. Okies. I'm ok with that", "blue");
-		visit_url("place.php?whichplace=beanstalk");
+		auto_log_warning("I see the beanstalk has already been planted. fixing questL10Garbage to step1", "blue");
+		set_property("questL10Garbage", "step1");
```

Comment:
If you want to force the property to that, fine. You raised the mafia issue and maybe someone will look into it but you still need the visit_url or you're going to break every single Ed run once this is merged.

That is unless mafia have finally fixed the issues Veracity stopped caring about in the bug report I linked on #459